gui/common/system.py

```diff
@@ -88,8 +88,32 @@ def get_sw_year():
     return str(datetime.now().year)
 
 
+def get_freenas_var_ds_config(f, var):
+    """
+    False gets picked up in get_freenas_var()
+    and results in default value (if it exists)
+    being set.
+    """
+    ds_config = []
```

Comment:
Is the file guaranteed to exist?

Reply:
It should be generated by this time, but there's not a strong guarantee. I'll add a check.